pypeit/inputfiles.py

```diff
@@ -698,24 +700,45 @@ def get_pypeitpar(self):
         # set of file names each time they are requested.  However, this should
         # only be done once in the code below because as soon as a relevant file
         # is found the loops are discontinued using `break`.
+        filenames = self.filenames.copy()
```

Comment:
Good call to save this instead of using the `property` method every time.  But do we need the `copy`?

Reply:
I think we don't need the `copy()`.  Since the `@property` returns a list, we can do whatever we need to with it, but `filenames` is just a list.